### PR TITLE
Add 🐙 userver

### PR DESCRIPTION
**Pull request by apolukhin**
### Pre-submission checklist:

*Please check each of these after submitting your pull request:*

* [x] Are you only including a `repo_url` if your project is 100% open source? If so, you need to pick the single best GitHub repository for your project, not a GitHub organization.
* [x] Is your project closed source or, if it is open source, does your project have at least 300 GitHub stars?
* [x] Have you picked the single best (existing) category for your project?
* [x] Does it follow the other guidelines from the [new entries](https://github.com/cncf/landscape#new-entries) section?
* [x] Have you added your SVG to `hosted_logos` and referenced it there?
* [x] Does your logo clearly state the name of the project/product and follow the other logo [guidelines](https://github.com/cncf/landscape#logos)?
* [x] Does your project/product name match the text on the logo?
* [x] Have you verified that the Crunchbase data for your organization is correct (including headquarters and LinkedIn)?
* [x] ~30 minutes after opening the pull request, the CNCF-Bot will post the URL for your staging server. Have you confirmed that it looks good to you and then added a comment to the PR saying "LGTM"?

### CODE REVIEW

**Comment by apolukhin** [comment]
LGTM

**Comment by apolukhin** [comment]
Hi,

@jeefy do I have to change something? I see no emoji usage in other projects. Are they forbidden?

**Comment by castrojo** [comment]
I'm ready to merge this but if we add emojis then we have to start caring about how it looks on the front end design wise, etc. 

If you wouldn't mind adjusting for plain text then this is ready to go, thanks!

(It is quite cute though!)

File: landscape.yml [hunk 1]
```diff
@@ -5417,6 +5417,42 @@ landscape:
...
+                creation of resilient C++ microservices, services and utilities.
+            homepage_url: https://userver.tech
+            repo_url: https://github.com/userver-framework/userver
+            logo: userver.svg
+            crunchbase: https://www.crunchbase.com/organization/yandex
+            extra:
+              summary_personas: Developers, Cloud Architects, Platform engineers, SREs, Network Engineers, Application Developers, Architects
+              summary_tags: Developer APIs, Distributed Systems, Streaming Data, PubSub, Microservices, HTTP, gRPC, Python, C++, CLI, observability, metrics, tracing, logs
+              summary_use_case: >-
+                🐙 userver solves the problem of efficient
```
Comment (castrojo) [inline landscape.yml:5434]:
```suggestion
                userver solves the problem of efficient
```

**Review by castrojo — CHANGES_REQUESTED** [review]

**Comment by apolukhin** [comment]
LGTM

**Review by castrojo — APPROVED** [review]

**Pull request merged**